src/main/java/seedu/recipe/model/recipe/ingredient/IngredientQuantifier.java

```diff
@@ -0,0 +1,73 @@
+package seedu.recipe.model.recipe.ingredient;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Represents a container for the values stored with respect to an Ingredient,
+ * within a Recipe instance.
+ */
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class IngredientQuantifier {
```

Comment:
Could the class name be `IngredientInformation` instead? Since quantifier brings the meaning of expressing quantity and we already have a separate `IngredientQuantity` class, might be misleading.

Reply:
Done